lib/mcb.rb

```diff
@@ -247,6 +247,9 @@ def iterate_v1_endpoint(url:, endpoint:, **opts)
     def remote_connect_options
       envs = env_to_azure_map.keys.join(', ')
       Proc.new do
+        option :R, 'recruitment_year',
+               "Set the recruitment year, defaults to 2019",
+               argument: :required
```

Comment:
reads kinda wrong...
argument being required yet defaulting?

Reply:
I think it's the `recruitment_year` value that's required.

So this is valid:

```
mcb providers list
mcb providers list -R 2020
```

But not this:

```
mcb providers list -R
```